include/lbann/layers/transform/concatenate.hpp

```diff
@@ -377,7 +377,7 @@ fp_compute() {
   dc::tensor::Concatenate(this->get_activations(0),
                           this->get_prev_activations(0),
                           this->get_prev_activations(1),
-                          El::GPUManager::Stream());
+                          hydrogen::cuda::GetDefaultStream());
```

Comment:
@benson31 Here is the spot where cuda is still used.

Reply:
...what else would we use?